Expensify/App
# Remove animations from the Popover menus

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/6121

### Tests
Tested All menus on the web.

### QA Steps
1. Go to staging.new.expensify.com and login.
2. Click the + icon in a Group DM text input and verify the popup menu.
3. Follow other menus as well.


### Tested On


### Screenshots


#### Web | Desktop


https://user-images.githubusercontent.com/24370807/142234892-06389ff9-7781-4afa-a6b9-dafba116a099.mp4


#### Mobile Web


https://user-images.githubusercontent.com/24370807/142239726-b9ac9dfc-2193-4097-a3bc-526385cdaebe.mp4


#### iOS


https://user-images.githubusercontent.com/24370807/142245110-060c22d9-6fd1-45c8-a0c9-79d812c00577.mp4


#### Android


https://user-images.githubusercontent.com/24370807/142238585-011fb836-23df-47c1-b3d2-e7496ddb78b5.mp4
